src/rust/endpoint-plugin/Cargo.toml

```diff
@@ -9,5 +9,5 @@ edition = "2018"
 [dependencies]
 grapl-graph-descriptions = { path="../graph-descriptions", version="*" }
 derive-dynamic-node = { path="../derive-dynamic-node" }
-uuid = "0.8.2"
-thiserror = "1.0.23"
+uuid = { version = "0", features=["v4"] }
```

Comment:
Version 0?

Reply:
I'd seen this in the graph merger, thought it was your doing: https://github.com/grapl-security/grapl/blob/main/src/rust/graph-merger/Cargo.toml#L49, figured I'd at least try to be more consistent. Shall we make them all "0.8"?